config/default.js

```diff
@@ -22,6 +22,9 @@ module.exports = {
       save: 'save',
       new: 'Save Revision',
     },
+    plugins: {
+      directory: [path.join(__dirname, '../input-plugins')],
```

Comment:
content-types uses globalconfig to get the plugins and I cannot think of a way to change globalconfig on init